frontend/packages/console-dynamic-plugin-sdk/src/api/flag.ts

```diff
@@ -0,0 +1,34 @@
+import { Map as ImmutableMap } from 'immutable';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { useSelector } from 'react-redux';
+import { featureReducerName } from '../constants';
+
+export const useFlag: UseFlag = (flag) =>
```

Comment:
Nit: I'd call this file `useFlag.ts` to stay consistent with other files containing React hooks.